boards.txt

```diff
@@ -253,3 +253,36 @@ gd_mbed_f4xx.menu.opt.ogstd.build.flags.optimize=-Og
 gd_mbed_f4xx.menu.opt.ogstd.build.flags.ldspecs=
 
 ################################################################################################
+# Maker: Keyboardio
+################################################################################################
+keyboardio_model_100.name=Keyboardio Model 100
+
+keyboardio_model_100.build.core=arduino
+keyboardio_model_100.build.board=keyboardio_model_100
+keyboardio_model_100.build.mcu=cortex-m4
+# I think this is the right number. It's 2**20. The previous definition was 2**18
+keyboardio_model_100.upload.maximum_size=1048576
+keyboardio_model_100.upload.maximum_data_size=98304
+
+# These two definitions set -D flags to the compiler to get the right code compiled for our MCU 
+keyboardio_model_100.build.series=GD32F30x
+keyboardio_model_100.build.product_line=GD32F30X_XD
+
+keyboardio_model_100.build.variant=keyboardio_model_100
+keyboardio_model_100.upload.openocd_script=target/stm32f1x.cfg
+
+
+keyboardio_model_100.menu.upload_method.dfuUtilMethod=dfu-util (DFU - Keyboardio DAPBoot bootloader)
+keyboardio_model_100.menu.upload_method.dfuUtilMethod.upload.protocol=dfu
+keyboardio_model_100.menu.upload_method.dfuUtilMethod.upload.tool=dfu-util
+keyboardio_model_100.menu.upload_method.dfuUtilMethod.upload.options=
+keyboardio_model_100.menu.upload_method.dfuUtilMethod.build.flash_offset=0x2000
```

Comment:
This section defines a flash offest of `0x2000` which will, through the `platform.txt` using it in `LD_FLASH_OFFSET` and `VECT_TAB_OFFSET`. However this the PR for the PlatformIO board definition has no `offset_address` field in `upload` (aka it's equivalent 0x0). We will need to equalize that later.

Reply:
Merging for now to be able to test stuff more easily, will open issues for the rest.